src/lib/y2network/sysconfig/connection_config_writers/base.rb

```diff
@@ -45,7 +45,9 @@ def write(conn)
           file.lladdr = conn.lladdress
           file.startmode = conn.startmode.to_s
           file.ifplugd_priority = conn.startmode.priority if conn.startmode.name == "ifplugd"
-          file.ethtool_options = conn.ethtool_options unless conn.ethtool_options.empty?
+          if conn.ethtool_options && !conn.ethtool_options.empty?
```

Comment:
hmm, I start worry that we start having it all over. Why simply not allowing nil and also assign at least `""` which is default?

Reply:
For me, a `nil` means that the key is not in the file and an empty string that it is there but with no value. Do we need to make such a distinction? Probably no, so we could simplify as you have proposed and do not write anything with empty values.